lib/email/sender.rb

```diff
@@ -239,6 +241,25 @@ def self.host_for(base_url)
 
     private
 
+    def add_attachments(post)
+      uploads = post.uploads
+      return unless uploads.present?
```

Comment:
Technically, this is not needed, since `.each` won't iterate over an empty array.

You coud rewrite this

``` ruby
uploads = post.uploads
return unless uploads.present?
uploads.each do |upload|
```

with this one line

``` ruby
post.uploads.each do |upload
```